SECURITY_GUIDANCE.md

```diff
@@ -260,6 +261,15 @@ visible-devices-envvar-when-unprivileged = true
 
 We recommend leaving these settings at the default values, which will enforce the requested NVIDIA GPU limits for unprivileged containers.
 
+### Limit use of NVIDIA GPU Time-Slicing
+Bottlerocket supports NVIDIA GPU time-slicing, enabling system administrators to allocate a set of replicas for a GPU, which can be assigned to individual pods for running various workloads.
+
+Internally, GPU time-slicing is used to multiplex workloads from multiple replicas of the same underlying GPU, providing each process with an equal share of time across all pods. However, while time-slicing allows sharing of GPUs among multiple workloads, it does not provide memory or fault isolation between replicas.
+
+Without memory isolation, all workloads share the same memory space, which can lead to contention, and possible interference and/or unintended resource access between processes. This can affect the predictability and performance of each workload, as one process may consume more resources than anticipated, leaving less available for others. Moreover, without fault isolation, if one process fails, it has the potential to affect all other processes running on the GPU.
```

Comment:
The important part seems to be that this feature creates a channel that breaks the isolation implied by containers. Let's call that out directly so the security relevance is clear.

Also, I would avoid "and/or" since that creates some ambiguity when parsing the sentence.

```suggestion
Without memory isolation, all workloads share the same memory space, which can lead to contention, possible interference, and unintended resource access between processes. This can affect the predictability and performance of each workload, as one process may consume more resources than anticipated, leaving less available for others. Moreover, without fault isolation, if one process fails, it has the potential to affect all other processes running on the GPU. These impacts can happen to processes running in different containers with different privilege levels.
```

Reply:
Thanks, for catching this and the suggestion! 

I meant for this to be:
>and possible interference between processes.

Instead of:
>and possible interference and/or unintended resource access between processes